lua/wikis/leagueoflegends/MainPageLayout/data.lua

```diff
@@ -34,6 +34,12 @@ local CONTENT = {
 		padding = true,
 		boxid = 1504,
 	},
+	liquipediaApp = {
+		heading = 'Download the Liquipedia App',
+		padding = true,
+		body = '{{Liquipedia:App}}',
+		boxid = 1513,
```

Comment:
Use the same boxid across all wikis for the same content

Reply:
Any id/number that's ideally used? >_>